tests/unittests/tests-ztimer/tests-ztimer-mock.c

```diff
@@ -199,6 +199,10 @@ static void test_ztimer_mock_set16(void)
     ztimer_clock_t *z = &zmock.super;
 
     ztimer_mock_init(&zmock, 16);
+
+    /* make sure ztimer stays turned on */
+    ztimer_acquire(z);
```

Comment:
I don't quite get why this is placed here for this particular test, why there is no corresponding release, and why seemingly all other tests don't need a `ztimer_acquire()`. I didn't read the test code, though. It just looks to me like a work-in-progress state here.

Reply:
This one is required since the test case makes use of `ztimer_now()`. And `ztimer_now()` requires a precedent `ztimer_acquire()` when `ztimer_ondemand` is active ;-)